lib/properties/borderSpacing.js

```diff
@@ -10,6 +10,9 @@ var parse = function parse(v) {
     if (v === '' || v === null) {
         return undefined;
     }
+    if (typeof v === 'number') {
+        v = String(v)
```

Comment:
I think `return undefined` makes more sense to mimic the current browser behaviour.